provider/vdpa_sim_blk_utils.py

```diff
@@ -0,0 +1,79 @@
+"""
+Module for VDPA block device interfaces.
+"""
+import logging
+from avocado.utils import process
+
+from virttest.utils_kernel_module import KernelModuleHandler
+
+LOG = logging.getLogger('avocado.test')
+
+
+class VDPABlkSimulatorError(Exception):
+    """ General VDPA BLK error"""
+    pass
+
+
+class VDPABlkSimulatorTest(object):
```

Comment:
Hi @qingwangrh 
I think we can inherit the `VDPASimulatorTest` https://github.com/avocado-framework/avocado-vt/blob/49e33beadd26651f184ab7ad4ae114c3a1320cf5/virttest/utils_vdpa.py#L205
then override the `setup` method and add the `add_vdpa_blk_dev`, `remove_vdpa_blk_dev` methods according to the `blk simulator` requirements.
CC @XueqiangWei